common/src/thrift/tachyon.thrift

```diff
@@ -147,15 +108,15 @@ service BlockMasterService {
 
   void workerRegister(1: i64 workerId, 2: list<i64> totalBytesOnTiers,
       3: list<i64> usedBytesOnTiers, 4: map<i64, list<i64>> currentBlocksOnTiers)
-    throws (1: TachyonException te)
+    throws (1: TachyonTException e)
```

Comment:
Would it make sense to always throw both TachyonTException and TachyonIOException even if the latter might not be thrown? That way, the client and server side code that deals with RPCs can be uniform and won't need to change if an RPC method suddenly can throw an IOException.